### ISSUE TITLE
any help is appreciated.. 

### ISSUE THREAD

**Issue by ghost** [markwk/qs_ledger#21]
Hello
where is the error? it's the right path.. followed all the procedure..

![vvfsAJl](https://user-images.githubusercontent.com/60441912/161377838-f0c7af01-ac89-4b38-b5ee-181f8c593f05.png)

Thanks!

**Comment by noniguez** [comment]
Hey there! 

This script is not updated. Hence, it won't work. I recommend downloading the free iOS app HealthExportCSV. It is very easy to get your data in .csv directly.

Regards

> Hello where is the error? it's the right path.. followed all the procedure..
> 
> ![vvfsAJl](https://user-images.githubusercontent.com/60441912/161377838-f0c7af01-ac89-4b38-b5ee-181f8c593f05.png)
> 
> Thanks!

**Comment by ghost** [comment]
ok, thanks for the update.. but the app is 2 euro.. is not possible a version free?

**Comment by noniguez** [comment]
![image](https://user-images.githubusercontent.com/95111314/161797823-af30c7c7-f6f1-4b88-9381-a0c9c1fba76e.png)
It should be this red one, it should be free.

do not confuse with a green app

**Comment by ghost** [comment]
thanks I found it and its working.. but after the exporting there is a method how to view the data and maybe create a graphic? 
thank you

**Comment by noniguez** [comment]
> thanks I found it and its working.. but after the exporting there is a method how to view the data and maybe create a graphic? thank you

Yes, you need this Jupyter file. Run it and you'll get some nice graphs.

https://github.com/markwk/qs_ledger/blob/master/apple_health/apple_health_data_processor.ipynb
